javascript/big-practice/src/scripts/views/listNoteView.js

```diff
@@ -103,5 +189,43 @@ export default class ListNoteView {
     this.closeButtonElement.addEventListener('click', () => {
       this.addNewNote(handler);
     });
-  };
+  }
+
+  /**
+   * @description function event delete note
+   *
+   * @param {function} handler is function delete transmitted from from the model
+   */
+  static bindDeleteNotes(handler) {
+    const deleteButtonElements = selectDOMClassAll('.btn-delete');
+
+    deleteButtonElements.forEach((btn) => {
+      btn.addEventListener('click', (element) => {
+        const noteId = element.target.getAttribute('id');
+        const noteElement = selectDOMClassAll('.note');
+
+        noteElement.forEach((note) => {
+          if (note.id === noteId) {
+            note.remove();
+            handler(noteId);
+          }
+        });
+      });
+    });
+  }
+
+  /**
+   * @description function delete selected notes
+   * @param {function} handler is function delete transmitted from from the model
+   */
+  bindDeleteListNotes(handler) {
+    this.btnDeleteBulkActions.addEventListener('click', () => {
+      const noteSelected = selectDOMClassAll('.selected');
+
+      noteSelected.forEach((note) => {
```

Comment:
Update UI here not correct, I think you should update the data then re-render the list. What if the data not update correctly and the UI change by itself, our app will not consistent